vochain/scrutinizer/vote.go

```diff
@@ -265,6 +266,7 @@ func (s *Scrutinizer) ComputeResult(processID []byte) error {
 	if err := s.db.Update(processID, p); err != nil {
 		return fmt.Errorf("computeResult: cannot update processID %x: %w, ", processID, err)
 	}
+	s.processUpdateMutex.RUnlock()
```

Comment:
This must be `defer s.processUpdateMutex.RUnlock` on line 253, since there is a return in the middle